README.md

```diff
@@ -60,7 +60,12 @@ Once you have implemented a lower bound search for the start of the range, imple
 
 *How do you use binary search to find the lower bound of a number? How did you have to modify the binary search algorithm?*
 
+We search with binary splits to iterate outward from the middle value to find value v. If the middle value x[mid] != v then we iterate the middle value 1 towards v, and set this middle value to high or low depending on whether x[mid] is higher than or lower than v. We repeat this until we hit the value v with the binary splits, and then we keep moving our lower bound such that our high and low value will meet at the first v value found in the sorted list.
+ 
 *Would anything be more difficult if the features covered ranges instead of single nucleotides (like real BED files)? What could go wrong, if anything?*
 
+Yes it would make it more difficult. It would mean that the present comparison of only chrom index and chrom start would not nr sufficient for sorting the chromosomes. Overlaps in chrom 'start' and 'end' interval would occur. If we wanted a completely sorted and merged outfile we would have to take both chrom 'start' and chrom 'end' into account in the merging.
```

Comment:
Not so much the merging. The problem is that you have coordinates you could sort by, and even if those are sorted, _neither_ will help you catch cases like a large interval that starts way before the query interval and ends long after. Finding overlapping intervals is a different beast than finding single points in an interval.